search/indico_search/client/main.scss

```diff
@@ -169,11 +169,9 @@
 
 .search-tag {
     max-width: 112px;
-    display: table-cell;
+    display: flexbox;
```

Comment:
isn't that the legaxy flexbox (vs `display: flex` for the current one)?

Reply:
True. Fixed.